model/internal/pdata/common_test.go

```diff
@@ -976,17 +976,17 @@ func generateTestBytesMap() Map {
 	})
 }
 
-func TestAttributeValueArray(t *testing.T) {
-	a1 := NewValueArray()
-	assert.EqualValues(t, ValueTypeArray, a1.Type())
+func TestAttributeValueSlice(t *testing.T) {
```

Comment:
Separate PR consider to remove `Attribute` from test names.

Reply:
Sounds good. I'll do